src/ugeneui/html/version_news_en.html

```diff
@@ -18,14 +18,34 @@
  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
  MA 02110-1301, USA.
 -->
-<h3>Changes in UGENE v. 39</h3>
+<h3>Changes in UGENE v. 40</h3>
+
+<h4>Alignment Editor</h4>
+<ul>
+    <li style="margin-bottom: 10px;"> Multi-row selection support.</li>
+    <li style="margin-bottom: 10px;">Built Tree dialog now has all PhyML algorithms listed.</li>
+    <li style="margin-bottom: 10px;">A new option to move selected sequences from one document to another.</li>
```

Comment:
A new option **of selected sequences moving**  from one document to another.

Reply:
Both variants "A new option to move" or "A new option of moving" are good. "of selected sequence moving" sounds very unnatural so.
I prefer the first one, it is more widely used. Compare:  counts of results for 
* https://www.google.com/search?q=%22A+new+option+to+pay%22
vs
* https://www.google.com/search?q="A+new+option+of+paying"